.github/workflows/snyk-issue.yml

```diff
@@ -0,0 +1,29 @@
+name: Snyk Issue
+
+on:
+  schedule:
+    - cron: '* */12 * * *'
+
+concurrency: snyk-issue
+
+jobs:
+  whitesource:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout action
+      uses: actions/checkout@v2
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.WHITESOURCE_ACTION_TOKEN }}
+        path: whitesource-actions
+    - name: set-env
```

Comment:
Please be consistent with these names, some of them are all lower case and some of them aren't